controller_manager/src/controller_manager.cpp

```diff
@@ -1266,21 +1279,21 @@ controller_interface::return_type ControllerManager::switch_controller(
     // if controller is not inactive then do not do any following-controllers checks
     if (!is_controller_inactive(controller_it->c))
     {
-      RCLCPP_WARN(
-        get_logger(),
-        "Controller with name '%s' is not inactive so its following "
-        "controllers do not have to be checked, because it cannot be activated.",
-        controller_it->info.name.c_str());
+      message = "Controller with name '" + controller_it->info.name +
+                "' is not inactive so its following controllers do not have to be checked, because "
+                "it cannot be activated.";
+      RCLCPP_WARN(get_logger(), "%s", message.c_str());
```

Comment:
This message confused me, there is no "following controller" in a simple setup (example_1).

```
$ ros2 control switch_controllers --activate joint_trajectory_position_controller --strict
Controller with name 'joint_trajectory_position_controller' is not inactive so its following controllers do not have to be checked, because it cannot be activated.
```

Reply:
I believe it could be a case where the controller is connected to any chainable controller. This is a method that checks it.

Reply:
I agree!

Reply:
Can you check now?